### PR TITLE
Fixing and updating QA plan

### PR DESCRIPTION
**Pull request by mpoke**
Fixes some changes introduced in https://github.com/cosmos/interchain-security/pull/363

Updates some entry.

### CODE REVIEW

**Comment by mpoke** [comment]
@danwt , @sainoe , @smarshall-spitzbart could you please fill in the placeholders (either TODOs or your names)?

File: docs/quality_assurance.md [hunk 1]
```diff
@@ -73,27 +72,27 @@ The main concern addressed in this section is the correctness of the provider ch
...
+| 4.05 | Unbonding operations (undelegations, redelegations, validator unbondings) should eventually complete even if one of the clients expire <br /> - expected outcome: the pending VSC packets eventually timeout, which leads to the consumer chain removal <br /> - requires https://github.com/cosmos/interchain-security/issues/283 | `Scheduled` | `NA` | `Scheduled` | `Future work` | `Scheduled` | `NA` |
+| 4.06 | A validator cannot get slashed more than once for double signing, regardless of how many times it double signs on different chains (consumers or provider) | `Scheduled` | `NA` |`Done` <br /> see [TestHandleSlashPacketErrors](../tests/e2e/slashing_test.go#L317) | `Done` | `Scheduled` | `NA` |
+| 4.07 | A validator cannot get slashed multiple times for downtime on the same consumer chain without requesting to `Unjail` itself on the provider chain in between | `Scheduled` | `NA` | `Done` <br /> (TODO: link) | `Partial coverage` | `Scheduled` | `NA` |
```
Comment (smarshall-spitzbart) [inline docs/quality_assurance.md:81]: [TestSendSlashPacket](https://github.com/cosmos/interchain-security/blob/main/tests/e2e/slashing_test.go#L648) is the closest I can find for this, but lacks the unjailing part I believe

**Review by smarshall-spitzbart — COMMENTED** [review]
These changes all look good to me 👍. Note that I kept the "here"s for links so that the table would fit on my screen without scrolling when rendered, but I do see value in specifying test names

File: docs/quality_assurance.md (hunk 1 above)
Comment (mpoke) [reply docs/quality_assurance.md:81]: Partial coverage then?

Comment (smarshall-spitzbart) [reply docs/quality_assurance.md:81]: Yea that seems reasonable

File: docs/quality_assurance.md [hunk 2]
```diff
@@ -73,27 +72,27 @@ The main concern addressed in this section is the correctness of the provider ch
...
-| 4.11 | The provider chain can graciously handle a CCV packet timing out (without shuting down) <br /> - expected outcome: consumer chain shuts down and its state in provider CCV module is removed | `Scheduled` | `??` | `Future work` | `Future work` | `Scheduled` | `NA` |
-| 4.12 | The provider chain can graciously handle a `ConsumerRemovalProposal` <br /> - expected outcome: consumer chain shuts down and its state in provider CCV module is removed | `Scheduled` | `Done` <br /> see [here](../x/ccv/provider/keeper/proposal_test.go#L313) | `NA` | `Future work` | `Scheduled` | `NA` |
-| 4.13 | The provider chain can graciously handle a `ConsumerAdditionProposal` <br /> - expected outcome: a consumer chain is registered and a client is created | `Scheduled` |`Done` <br /> see [here](../x/ccv/provider/keeper/proposal_test.go#L31) | `NA` | `Future work` | `Scheduled` | `NA` |
+| 4.09 | The provider chain can easily be restarted with IS enabled <br /> - `ExportGenesis` & `InitGenesis` | `Scheduled` | `??` (Simon?) | `Future work` (Simon?) | `Future work` | `Scheduled` | `NA` |
```
Comment (sainoe) [inline docs/quality_assurance.md:83]:
```suggestion
| 4.09 | The provider chain can easily be restarted with IS enabled <br /> - `ExportGenesis` & `InitGenesis`  <br /> - requires https://github.com/informalsystems/hermes/issues/1152| `Scheduled` | `Done` <br /> [TestInitAndExportGenesis](../x/ccv/provider/keeper/genesis_test.go#L20) | `Future work` | `Future work` | `Scheduled` | `NA` |
```

**Comment by mpoke** [comment]
I'm merging these w/o approvals since it's only docs.

**Pull request merged**